sklearn/datasets/_twenty_newsgroups.py

```diff
@@ -391,19 +393,38 @@ def fetch_20newsgroups_vectorized(*, subset="train", remove=(), data_home=None,
 
         .. versionadded:: 0.22
 
+    as_frame : bool, default=False
+        If True, the data is a pandas DataFrame including columns with
+        appropriate dtypes (numeric, string, or categorical). The target is
+        a pandas DataFrame or Series depending on the number of
+        `target_columns`.
+
+        .. note::
+            Since the dataset is sparse, loading it as a pandas DataFrame
```

Comment:
Theoretically one could still get the data in dense format, but I guess it's not really practical for anybody.